release-repo-scripts/bump_dependency.bash

```diff
@@ -455,8 +455,29 @@ for ((i = 0; i < "${#LIBRARIES[@]}"; i++)); do
     DEP_VER=${VERSIONS[$j]}
     DEP_PREV_VER="$((${DEP_VER}-1))"
 
+    # Rule: *plugin2 -> *plugin3
+    # Replace lines like: "find_package(ignition-cmake2)"
+    #               with: "find_package(ignition-cmake3)"
     find . -type f ! -name 'Changelog.md' ! -name 'Migration.md' -print0 | xargs -0 sed -i "s ${DEP_LIB}${DEP_PREV_VER} ${DEP_LIB}${DEP_VER} g"
 
+    # Rule: IGN_PLUGIN_VER 2 -> IGN_PLUGIN_VER 3
+    # Replace lines like: "set(IGN_PLUGIN_VER 2)"
+    #               with: "set(IGN_PLUGIN_VER 3)"
+    find . -type f ! -name 'Changelog.md' ! -name 'Migration.md' -print0 | xargs -0 sed -i "s@IGN_${DEP_LIB}_VER ${DEP_PREV_VER}@\UIGN_${DEP_LIB}_VER ${DEP_VER}@ig"
```

Comment:
I commented in the issue that I think `set(IGN_PLUGIN_VER 2)` is a bad pattern, and that we should replace this in the code with `set(IGN_PLUGIN_VER ${ignition-plugin2_VERSION_MAJOR})`

Reply:
Should it get bumped up to set(IGN_PLUGIN_VER ${ignition-plugin**3**_VERSION_MAJOR}) in those cases?

Reply:
https://github.com/ignition-tooling/release-tools/pull/700